Updating the team sync for org docs to include more detail about ente… (#23540)
github/docs: content/organizations/managing-saml-single-sign-on-for-your-organization/managing-team-synchronization-for-your-organization.md

{% data reusables.identity-and-permissions.sync-team-with-idp-group %}

> You can also enable team synchronization for all organizations owned by an enterprise account. If SAML is set up at the enterprise level, you cannot enable team sync on an individual organization. In that case, team sync must be enabled for the entire enterprise. For more information, see "[Managing team synchronization for organizations in your enterprise](/enterprise-cloud@latest/admin/authentication/managing-identity-and-access-for-your-enterprise/managing-team-synchronization-for-organizations-in-your-enterprise)."

Reviewer: ```suggestion
You can also enable team synchronization for all organizations owned by an enterprise account. If SAML is configured at the enterprise level, you cannot enable team synchronization on an individual organization. Instead, you must configure team synchronization for the entire enterprise. For more information, see "[Managing team synchronization for organizations in your enterprise](/enterprise-cloud@latest/admin/authentication/managing-identity-and-access-for-your-enterprise/managing-team-synchronization-for-organizations-in-your-enterprise)."
```
